microsoft-edge/devtools-guide-chromium/customize/index.md

```diff
@@ -88,12 +88,12 @@ Navigate to [Microsoft Edge DevTools Placement][DevToolsPlacement].
    Undocked DevTools  
 :::image-end:::  
 
-## Dark theme  
+## Theme  
 
-Navigate to [Enable Dark Theme][DarkTheme].  
+Navigate to [Apply color themes to Microsoft Edge DevTools][Theme].  
 
-:::image type="complex" source="../media/customize-settings-appearance-theme.msft.png" alt-text="The dark theme" lightbox="../media/customize-settings-appearance-theme.msft.png":::
-   The dark theme  
+:::image type="complex" source="./media/customize-theme-setting.png" alt-text="Seleting a different color theme" lightbox="./media/customize-theme-setting.png":::
+   Selecting a different color theme  
 :::image-end:::  
```

Comment:
Change:
You may move tools between the main panel and the drawer.
to:
You can move tools between the main panel and the drawer.
Per "can, may" article at MWSG, "do not use 'may'".  Mgmt confirmed.